src/components/ui/Report.svelte

```diff
@@ -169,11 +169,18 @@
   <p>{TRANSLATED.TEXT_NO_SAMPLE}</p>
 {/if}
 
-<h2>{TRANSLATED.HEADING_SPECIFICS}</h2>
+<h2>{TRANSLATED.LABEL_TECH}</h2>
 <p>{@html marked(report.specifics) || TRANSLATED.LABEL_NOT_PROVIDED}</p>
 
-<h2>{TRANSLATED.HEADING_RESOURCES}</h2>
-<ul>
+<h2>{TRANSLATED.HEADING_SPECIFICS}</h2>
+{#if true}
+  <p>{report.tech}</p>
+{:else}
+  <p>{TRANSLATED.LABEL_NOT_PROVIDED}</p>
+{/if}
+
+<h2 class='strip'>{TRANSLATED.HEADING_RESOURCES}</h2>
```

Comment:
I'm probably missing something, but wonder why this section and the list underneath are stripped?